resources/views/livewire/newsletter-signup-form-bottom-modal.blade.php

```diff
@@ -0,0 +1,25 @@
+<livewire-vue-adaptor v-slot="lw">
+    <user-interaction-context v-slot="interaction">
+        <bottom-modal
+            :show="
+                interaction.maxScrolledPercent >= {{ $openOnScrolledPercent }}
+                && interaction.timeSpentSeconds >= 30
+            "
+            v-on:open="lw.call('onOpen')"
+            v-on:close="lw.call('onDismissed')"
+        >
+            <div class="row py-5">
+                <div class="visible-lg-block col-lg-1 col-lg-offset-2 text-right pl-0 pt-4">
+                    <svg viewBox="0 0 101 110" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <rect x="3" y="3" width="94.7368" height="104" stroke="black" stroke-width="6"/>
+                        <rect x="21.5" y="20" width="40" height="40" fill="black"/>
+                        <rect x="47.8652" y="72.7769" width="20" height="20" transform="rotate(-20 47.8652 72.7769)" fill="black"/>
+                    </svg>
+                </div>
+                <div class="col-lg-7 pr-0 pl-2">
```

Comment:
```suggestion
                <div class="col-lg-7">
```

otherwise it breaks the spacing on mobile
<img width="315" alt="Screenshot 2021-11-22 at 00 44 53" src="https://user-images.githubusercontent.com/2682941/142783949-c06c4fb5-690c-402f-8efd-75f942363b22.png">
:

Reply:
Good catch!

Reply:
Thank you again. Fixed: 
![image](https://user-images.githubusercontent.com/1374745/143391194-021622c9-d635-4556-af57-ff48d2389378.png)